runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -38,6 +38,11 @@ vim9script
 # The communication with gdb uses GDB/MI.  See:
 # https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI.html
 
+var DEBUG = false
```

Comment:
I think we can make this:
```suggestion
var DEBUG = get(g:termdebug_config, 'debug', false)
```

I change it while merging.

Reply:
What if the g:termdebug_config does not exist? I think the get will return an error...

Reply:
or may be not 🙈

Reply:
I am pretty sure I tried, but please double check. That is why it's a bit ugly the way I proposed, I couldn't come with a better form.